daprdocs/content/en/reference/components-reference/supported-bindings/redis.md

```diff
@@ -88,6 +88,55 @@ You can store a record in Redis using the `create` operation. This sets a key to
 
 An HTTP 204 (No Content) and empty body is returned if successful.
 
+- `get`
```

Comment:
@wxbty Add it to the list of operations above also. Also add `delete`

Reply:
The action list has got and delete added, what are you referring to?